test_conformance/spirv_new/spirv_asm/spv1.4/no_integer_wrap_decoration_fadd_int.spvasm32

```diff
@@ -0,0 +1,46 @@
+; SPIR-V
+; Version: 1.4
+; Generator: Khronos SPIR-V Tools Assembler; 0
+; Bound: 23
+; Schema: 0
+               OpCapability Addresses
+               OpCapability Linkage
+               OpCapability Kernel
+          %1 = OpExtInstImport "OpenCL.std"
```

Comment:
Seems unused.

Reply:
Yeah, it is.  I fixed these in the SPIR-V files I added for this test, but I didn't change any of the existing SPIR-V files, including those used to test the "no integer wrap decoration" via the extension.

I think this is pretty common because the SPIR-V LLVM Translator appears to be adding the OpExtInstImport pretty much unconditionally, so if you start with a file generated by the SPIR-V LLVM Translator you'd need to delete it manually, and I'm not in the habit of doing that (yet).

Reply:
Ah, I wasn't even aware the SPIR-V LLVM Translator adds the OpExtInstImport unconditionally!